src/test.js

```diff
@@ -604,7 +608,14 @@ function test( testName, callback ) {
 		callback: callback
 	} );
 
-	newTest.queue();
+	if(!Promise)
```

Comment:
I think you need `typeof Promise == "undefined"`. Otherwise you would get a `ReferenceError`.